.gitignore

```diff
@@ -10,7 +10,8 @@ bin/
 obj/
 
 # Common node modules locations
-/node_modules
+Node/node_modules
```

Comment:
It might be better to do this as a patch at the end, in case the whole generated block gets replaced again, but is not critical.